vent/menus/add.py

```diff
@@ -16,7 +16,7 @@ def create(self):
         self.repo.when_value_edited()
 
     def quit(self, *args, **kwargs):
-        self.parentApp.switchForm(None)
+        self.parentApp.switchForm('MAIN')
```

Comment:
Is there a way to reset those variables then without  switching to none? Perhaps I can switch to None and then switch to 'MAIN' after clearing the variables?